src/Umbraco.Infrastructure/Security/MemberUserStore.cs

```diff
@@ -110,7 +110,7 @@ public override Task<IdentityResult> CreateAsync(
             UpdateMemberProperties(memberEntity, user, out bool _);
 
             // create the member
-            _memberService.Save(memberEntity);
+            _memberService.Save(memberEntity, PublishNotificationSaveOptions.None);
```

Comment:
Should this one actually issue the `.Saving` notification and we should be able to cancel it?  So it's just the `Saved` one we'd want to supress.

Reply:
Good point, fixed 😁